cah/commands.go

```diff
@@ -41,6 +42,17 @@ func (p *Plugin) AddCommands() {
 
 			return nil, err
 		},
+		AutocompleteFunc: func(data *dcmd.Data) ([]*discordgo.ApplicationCommandOptionChoice, error) {
```

Comment:
This thing here needs a limiter, official doc says `Choices for STRING, INTEGER, and NUMBER types for the user to pick from, max 25` this is also same for ApplicaitonCommand.Options field....

I run into it because I have 27 CAH packs : )
but problem could be yeah even more common for rolemenu groups